src/main.tsx

```diff
@@ -1,10 +1,10 @@
-import { StrictMode } from 'react'
-import { createRoot } from 'react-dom/client'
-import './styles/main.scss'
-import App from './App.tsx'
+import { StrictMode } from "react";
+import { createRoot } from "react-dom/client";
+// import './styles/main.scss'
```

Comment:
this was required to be used in order to apply our design system. commenting this is incorrect and u should use this and styles provided by this in the app to follow the design system. like using color variables, spacing variables, mixins for media queries, the button styles, etc. add it back and update ur styles files as per the design system and comments made on them

Reply:
added